{"uid":"6b9212cf65f094fa","name":"test_deploy","fullName":"tests.integration.test_charm#test_deploy","historyId":"c20a6e30f9eb9eadd6b1ef7e9ad2158c","time":{"start":1730165438556,"stop":1730165637567,"duration":199011},"description":"Deploy the charm-under-test.\n\n    Assert on the unit status before any relations/configurations take place.\n    ","descriptionHtml":"<p>Deploy the charm-under-test.</p>\n<pre><code>Assert on the unit status before any relations/configurations take place.\n</code></pre>\n","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/test_charm.py\", line 59, in test_deploy\n    await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status=\"active\", timeout=1500)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/0","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1730165438399,"stop":1730165438400,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1730165438400,"stop":1730165438553,"duration":153},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1730165438400,"stop":1730165438400,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1730165438553,"stop":1730165438555,"duration":2},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1730165438556,"stop":1730165438556,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Deploy the charm-under-test.\n\n    Assert on the unit status before any relations/configurations take place.\n    ","status":"broken","statusMessage":"juju.errors.JujuUnitError: Unit in error: postgresql/0","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/test_charm.py\", line 59, in test_deploy\n    await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status=\"active\", timeout=1500)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2963, in wait_for_idle\n    _raise_for_status(errors, \"error\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2869, in _raise_for_status\n    raise error_type(\"{}{} in {}: {}\".format(\njuju.errors.JujuUnitError: Unit in error: postgresql/0","steps":[],"attachments":[{"uid":"8fd194a28a9a20b","name":"log","source":"8fd194a28a9a20b.txt","type":"text/plain","size":1854}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1730165637647,"stop":1730165637647,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1730165637675,"stop":1730165638255,"duration":580},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm::finalizer","time":{"start":1730165637675,"stop":1730165637675,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1730165638256,"stop":1730165638256,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1730165638256,"stop":1730165638257,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1730165638257,"stop":1730165638257,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"skip_if_deployed"},{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_charm"},{"name":"host","value":"fv-az979-438"},{"name":"thread","value":"15992-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_charm"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"5c2c33fb7f397a90","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":16,"skipped":0,"passed":125,"unknown":2,"total":143},"items":[{"uid":"79badc2a7a867fa7","reportUrl":"../218//#testresult/79badc2a7a867fa7","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1730079014898,"stop":1730079210277,"duration":195379}},{"uid":"56d61f5691385f02","reportUrl":"../217//#testresult/56d61f5691385f02","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729992631153,"stop":1729992817923,"duration":186770}},{"uid":"a54ed0dbac677279","reportUrl":"../216//#testresult/a54ed0dbac677279","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/2","time":{"start":1729906243535,"stop":1729906430939,"duration":187404}},{"uid":"436a7f18b8b1ef16","reportUrl":"../215//#testresult/436a7f18b8b1ef16","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729819849539,"stop":1729820037875,"duration":188336}},{"uid":"4733bc604ea8b53","reportUrl":"../214//#testresult/4733bc604ea8b53","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1729733780475,"stop":1729733975187,"duration":194712}},{"uid":"18548790cc0937f5","reportUrl":"../213//#testresult/18548790cc0937f5","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1729646998501,"stop":1729647184742,"duration":186241}},{"uid":"eaf482c173bfcf6e","reportUrl":"../212//#testresult/eaf482c173bfcf6e","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729560592948,"stop":1729560782355,"duration":189407}},{"uid":"b0fe9ac0514ef107","reportUrl":"../211//#testresult/b0fe9ac0514ef107","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1729474182865,"stop":1729474365930,"duration":183065}},{"uid":"a190817b59a0f48a","reportUrl":"../210//#testresult/a190817b59a0f48a","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/1","time":{"start":1729387827097,"stop":1729388012142,"duration":185045}},{"uid":"e2772716ff174467","reportUrl":"../209//#testresult/e2772716ff174467","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729301384351,"stop":1729301572549,"duration":188198}},{"uid":"66ef801d301d36f3","reportUrl":"../208//#testresult/66ef801d301d36f3","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729215027790,"stop":1729215216783,"duration":188993}},{"uid":"256d13813363a33a","reportUrl":"../207//#testresult/256d13813363a33a","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: postgresql/0","time":{"start":1729128699721,"stop":1729128904165,"duration":204444}},{"uid":"c23662a5e0bbc492","reportUrl":"../206//#testresult/c23662a5e0bbc492","status":"unknown","time":{}},{"uid":"d8df8e48fe7e1092","reportUrl":"../205//#testresult/d8df8e48fe7e1092","status":"passed","time":{"start":1728955823363,"stop":1728956407930,"duration":584567}},{"uid":"dae3472f8f310e7d","reportUrl":"../204//#testresult/dae3472f8f310e7d","status":"passed","time":{"start":1728869450103,"stop":1728870035306,"duration":585203}},{"uid":"47e1c0702d913a42","reportUrl":"../203//#testresult/47e1c0702d913a42","status":"passed","time":{"start":1728783007366,"stop":1728783585253,"duration":577887}},{"uid":"4dfe9216c4f38ed2","reportUrl":"../202//#testresult/4dfe9216c4f38ed2","status":"passed","time":{"start":1728696640056,"stop":1728697245040,"duration":604984}},{"uid":"f482f8876f6d60d1","reportUrl":"../201//#testresult/f482f8876f6d60d1","status":"passed","time":{"start":1728610164256,"stop":1728610733599,"duration":569343}},{"uid":"b9e9842d7589b7a6","reportUrl":"../200//#testresult/b9e9842d7589b7a6","status":"passed","time":{"start":1728523782072,"stop":1728524394205,"duration":612133}},{"uid":"5d06ef274fb1a884","reportUrl":"../199//#testresult/5d06ef274fb1a884","status":"passed","time":{"start":1728437435142,"stop":1728438022049,"duration":586907}}]},"tags":["skip_if_deployed","asyncio","abort_on_fail","group(1)"]},"source":"6b9212cf65f094fa.json","parameterValues":[]}